services.html

```diff
@@ -0,0 +1,113 @@
+<!doctype html>
+
```

Comment:
you're the first person I've seen make a service.html page for the DOM I assignment. even though it's just the code the great ideas web page I found this interesting and I like the creativity.